client/web/src/hooks/auth.ts

```diff
@@ -73,7 +84,7 @@ export default function useAuth() {
   useEffect(() => {
     loadAuth().then((d) => {
       if (
-        !d?.canManageNode &&
+        !d?.authorized &&
```

Comment:
I think we may also want to check the user's capabilities at this point.  If they have zero capabilities, then we know that they will never be authorized to manage any part of the client, so there's no point in sending them through check mode.  I think it would basically be the same `hasAnyPermissions` check you're doing in login-toggle.

Reply:
Oh good catch! Yeah incase they explicitly go to the "?check=now" URL (or just end up in a funky state when changing tailnets or something). Will update